modules/installation-configuration-parameters.adoc

```diff
@@ -2469,7 +2469,7 @@ When running your cluster on GCP 64-bit ARM infrastructures, ensure that you use
       osDisk:
         diskType:
 |The link:https://cloud.google.com/compute/docs/disks#disk-types[GCP disk type] for compute machines.
-|`pd-ssd`, `pd-standard`, or `pd-balanced`. The default is `pd-ssd`.
+|Valid values are `pd-ssd`, `pd-standard`, `hyperdisk-balanced`, or `pd-balanced`. The default value is `pd-ssd`.
```

Comment:
Suggest to list the valid types in the same order as the command `openshift-install explain`, thanks! 
```
$ openshift-install explain installconfig.compute.platform.gcp.osDisk.diskType
KIND:     InstallConfig
VERSION:  v1

RESOURCE: <string>
  Valid Values: "pd-balanced","pd-ssd","pd-standard","hyperdisk-balanced"
  DiskType defines the type of disk. For control plane nodes, the valid values are pd-balanced, pd-ssd, and hyperdisk-balanced.

$